UPGRADE.md

```diff
@@ -1,4 +1,24 @@
 # UPGRADING
+## Recommended way of upgrading
+* upgrade locally first. After you fix all issues caused by the upgrade, version them and then continue with upgrading application on a server
```

Comment:
I would not say "version them" - it seems like you should version all the issues...
Maybe use "caused by the upgrade, **commit your changes**...", instead?